src/app/credential-issuer/middleware.js

```diff
@@ -18,13 +18,16 @@ module.exports = {
     next();
   },
   sendParamsToAPI: async (req, res, next) => {
+    const { id } = req.query;
+
+    const criId = req.params.criId || id;
+    const redirectUri = ["dcmaw, stubDcmaw"].includes(criId)
```

Comment:
I'm gonna change this bit slightly. I think it will be better if I just have 2 different `sendParamsToAPI` methods, one for each callback URL. This will make the future transitions of the other CRI's a bit easier